kong/llm/schemas/init.lua

```diff
@@ -237,12 +237,6 @@ return {
     { logging = logging_schema },
   },
   entity_checks = {
-    -- these three checks run in a chain, to ensure that all auth params for each respective "set" are specified
-    { conditional_at_least_one_of = { if_field = "model.provider",
-                                      if_match = { one_of = { "openai", "azure", "anthropic", "cohere" } },
-                                      then_at_least_one_of = { "auth.header_name", "auth.param_name" },
-                                      then_err = "must set one of %s, and its respective options, when provider is not self-hosted" }},
-
     { mutually_required = { "auth.header_name", "auth.header_value" }, },
```

Comment:
Do we still need these two checkers?
I'm not sure if the checkers will fail when neither header_name nor param_name is provided.

Reply:
```
  MUTUALLY_REQUIRED         = "all or none of these fields must be set: %s",
```
Yeah, it will be failed when neither header_name nor param_name is provided. It's our case needs to be checked. So it 's Okay for the current case.